ng-dev/release/stamping/env-stamp.ts

```diff
@@ -28,6 +28,7 @@ export function buildEnvStamp(mode: EnvStampMode) {
   console.info(`BUILD_SCM_BRANCH ${getCurrentBranch(git)}`);
   console.info(`BUILD_SCM_COMMIT_SHA ${getCurrentSha(git)}`);
   console.info(`BUILD_SCM_HASH ${getCurrentSha(git)}`);
+  console.info(`BUILD_SCM_HASH_ABBREV ${getCurrentAbbrevSha(git)}`);
```

Comment:
Debating between `HASH_ABBREV`, `ABBREV_HASH`, `SHORT_HASH`. Wdyt?

Reply:
Lets go with `ABBREV_HASH` actually since that matches the git documentation.